trigger/host.go

```diff
@@ -70,17 +77,19 @@ func (t *hostBase) Attributes() event.Attributes {
 }
 
 type hostTemplateData struct {
-	ID             string
-	Name           string
-	Distro         string
-	ExpirationTime string
-	URL            string
+	ID                    string
+	Name                  string
+	Distro                string
+	ExpirationTime        string
+	LastCommunicationTime string
```

Comment:
nit: would call this something other than last communication time (maybe `IdleSince`?). That would avoid overloading the meaning of communication time and confusing future readers perusing this logic. Not that big of a deal though.